File: Java/Program/Clothing.java
class Clothing extends Product {
    private String size;     //String untuk size
    private String material; //String untuk material
    private String gender;   //String untuk gender

    //Konstruktor deafault untuk class clothing
    public Clothing() {
        this.size = "";
        this.material = "";
        this.gender = "";
    }

    //Method untuk set size
    public void setSize(String size) {
        this.size = size;
    }
    //method untuk set gender
    public void setGender(String gender) {
        this.gender = gender;
    }
    //Method untuk set material
    public void setMaterial(String material) {
        this.material = material;
    }
    //Method untuk get size
    public String getSize() {
        return size;
    }
    //Method untuk get material
    public String getMaterial() {
        return material;
    }
    //Method untuk get gender
    public String getGender() {
        return gender;
    }
}

File: CPP/Program/Clothing.cpp
#include <string.h>
#include "Product.cpp"

class Clothing : public Product {
private:
    string size;     // String untuk size
    string material; // String untuk material
    string gender;   // String untuk gender

public:
    // Konstruktor deafault untuk class clothing
    Clothing() {
        size = "";
        material = "";
        gender = "";
    }

    // Method untuk set size
    void setSize(string size) {
        this->size = size;
    }

    // method untuk set gender
    void setGender(string gender) {
        this->gender = gender;
    }

    // Method untuk set material
    void setMaterial(string material) {
        this->material = material;
    }

    // Method untuk get size
    string getSize() {
        return size;
    }

    // Method untuk get material
    string getMaterial() {
        return material;
    }

    // Method untuk get gender
    string getGender() {
        return gender;
    }
};